test/integration/annotation_spec.js

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2020 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("Annotation highlight", () => {
+  describe("annotation-highlight.pdf", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await Promise.all(
+        global.integrationSessions.map(async session => {
+          const page = await session.browser.newPage();
+          await page.goto(
+            `${global.integrationBaseUrl}?file=/test/pdfs/annotation-highlight.pdf`
+          );
+          await page.bringToFront();
+          await page.waitForSelector("[data-annotation-id='19R']", {
+            timeout: 0,
+          });
+          return page;
+        })
+      );
+    }, 1e5);
```

Comment:
Having a *seemingly* arbitrary constant here, and in other files, is neither very clear nor maintainable in the future. Please model the new `test/integration-boot.js` file after the "boot"-files used with the existing unit- and font-tests instead, note in particular https://github.com/mozilla/pdf.js/blob/93b3ba2abb4e2eb760589899ff38cb99e8f0cbae/test/unit/jasmine-boot.js#L195-L196 and
https://github.com/mozilla/pdf.js/blob/93b3ba2abb4e2eb760589899ff38cb99e8f0cbae/test/font/jasmine-boot.js#L136-L137

Reply:
I tried to play with `jasmine.DEFAULT_TIMEOUT_INTERVAL` and it never worked.
Finally I found how to set it correctly: `jasmine.jasmine.DEFAULT_TIMEOUT_INTERVAL`.